src/main/java/com/google/api/generator/gapic/composer/ServiceClientSampleCodeComposer.java

```diff
@@ -649,6 +693,136 @@ public static String composePagedCallableMethodHeaderSampleCode(
     return bodyExprs.stream().map(e -> ExprStatement.withExpr(e)).collect(Collectors.toList());
   }
 
+  private static List<Statement> composeClientStreamMethodSampleCodeBodyStatements(
+      Method method,
+      VariableExpr clientVarExpr,
+      VariableExpr requestVarExpr,
+      AssignmentExpr requestAssignmentExpr) {
+    // Create responseObserver variable expression.
+    // e.g. ApiStream<EchoResponse> responseObserver
+    TypeNode responseObserverType =
+        TypeNode.withReference(
+            ConcreteReference.builder()
+                .setClazz(ApiStreamObserver.class)
+                .setGenerics(method.inputType().reference())
+                .build());
+    VariableExpr responseObserverVarExpr =
+        VariableExpr.withVariable(
+            Variable.builder().setName("responseObserver").setType(responseObserverType).build());
+    // Create Anonymous Class contains methods: onNext, onError, onCompleted
+    // TODO (summerji): <pre> and {@code} is not supporting rendering '@' character, it is evaluated
```

Comment:
Consider fixing this first before this PR. It's orthogonal enough that it doesn't conflict with the sample code logic in general, and it will allow us to generate the right thing with this PR.